test/codegen-generated-classes-test/src/main/resources/codegen-resources/multiauth/customization.config

```diff
@@ -0,0 +1,4 @@
+{
+    "skipEndpointTestGeneration": true,
+     "useMultiAuth": true
```

Comment:
Discussed offline, we should consider removing useMultiAuth

Reply:
Added `JAVA-8023` and also added in planning